WooCommerce/Classes/ViewRelated/ReusableViews/TitleAndSubtitleAndStatusTableViewCell.swift

```diff
@@ -37,8 +37,7 @@ final class TitleAndSubtitleAndStatusTableViewCell: UITableViewCell, SearchResul
 extension TitleAndSubtitleAndStatusTableViewCell {
     struct ViewModel: Hashable {
         /// A unique ID to avoid duplicated identifier for the view model in diffable datasource
-        private let id = UUID()
-
+        var id: String = UUID().uuidString
```

Comment:
Nit: I think we don't have to explicitly specify the type here.

Reply:
I'd like to keep the type declaration here to be consistent with the other properties 😅 Also it's clearer to see the type than having to interpret the default value (it's the same for the compiler as well). Does that make sense?